docs/reference/dscr.rst

```diff
@@ -0,0 +1,8 @@
+Discriminability
+****************
+
+.. currentmodule:: mgc.dscr
```

Comment:
not loving the name `dscr` could you just write out `discriminability`? or `discrim`?

Reply:
+1 to `discrim`